apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
```

Comment:
just a question about the filename, `submission` are you actually submitting? or are you transforming data? perhaps it could be called transform-data?

Reply:
it's the submission data for the model so it's the correct name yep

Reply:
i'd call it submission-data otherwise it's a bit vague but meh for now